Automattic/jetpack
# VideoPress: Remove X-18 rating option from videos

Automattic/greenhouse#954

#### Changes proposed in this Pull Request:

This PR removes the X (X-18) option from VideoPress videos, as it's not in line with our TOS.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a Jetpack site with a VideoPress plan enabled
* Before pulling this PR, upload a video and set its rating to X (X-18)
* Pull this PR
* Go to your media library (/wp-admin/upload.php)
* Select the X-18 video
* ✅ It should now appear as R (R-17)
* ✅ There must be no option to set it as X-18
* Create a new post
* Add a video block (/video)
* Upload a video or select one from your media library (uploaded with VideoPress)
* ✅ There must be no option to set the video as X-18 in the rating options
